fleet/lib/mimir/values.yaml

```diff
@@ -100,25 +102,37 @@ ingester:
   zoneAwareReplication:
     topologyKey: kubernetes.io/hostname
 querier:
-  replicas: 1
+  replicas: 2
```

Comment:
it seems 2 is the default for replicas

Reply:
I think we should set it explicitly as some of the components default to 1 replica for no particular reason and it seems like the chart defaults could either be normalized to not be HA or to be all HA and we don't know which (if any) will happen. E.g.: https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/values.yaml#L1946

Reply:
that works for me, I just wanted to make sure that was the intention. Thank you.